src/Access/AccessCheck.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Drupal\itk_pretix\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\node\NodeInterface;
+use Drupal\user\Entity\User;
+
+/**
+ * Access check.
+ */
+class AccessCheck implements AccessInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function access(AccountInterface $account, RouteMatchInterface $routeMatch) {
+    $node = $routeMatch->getParameter('node');
+    if ($node instanceof NodeInterface && $account->isAuthenticated()) {
+      $user = User::load($account->id());
+      if ($node->access('update', $user)) {
+        return AccessResult::allowed();
```

Comment:
Not sure what the access rules are in the business domain, but: 
- why not use `my_module.permissions.yml` to set up proper permissions?
- this doesn't seem in sync with the access control for downloading files in `Manager->fileDownload()`

Reply:
For now, any user that can update a node can run exporters and download the results. We may have to refine and improve this in the future.

See my comment https://github.com/itk-dev/itk_pretix_d8/pull/28#discussion_r533205105